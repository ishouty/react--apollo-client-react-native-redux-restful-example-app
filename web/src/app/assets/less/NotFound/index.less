.page-not-found-container {
  font-family: @font;
  margin: 0px auto;
  text-align: center;

  i {
    font-size: 10em;
  }

  h3 {
    font-weight: 400;
    padding: 20px 0px;
  }

  color: @color-dark-gray;

  a {

    color: @color-light-blue;

  }

  .main-container {
    margin-top: 10%;

    text-align: center !important;
  }

}