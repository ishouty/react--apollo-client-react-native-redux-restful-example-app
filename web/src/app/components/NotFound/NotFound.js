import React, { Component } from 'react'
import { PageNotFoundText } from '../../constants/text/text'
import { Row } from 'react-bootstrap'


export default class NotFound extends Component {

    render() {

        return (
            <div>

                <div className="page-not-found-container">

                    <Row className="main-container">

                        <i className="fa fa-frown-o" aria-hidden="true"></i>

                        <h3>{ PageNotFoundText.message }</h3>

                    </Row>

                </div>
            </div>
        )

    }

}