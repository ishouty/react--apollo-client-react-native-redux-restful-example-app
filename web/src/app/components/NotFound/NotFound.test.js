import React, {Component} from "react";
import {shallow} from "enzyme"
import toJson from "enzyme-to-json"
import NotFound from "./NotFound"


describe('NotFound', ()=> {

    test('Component should render match snap shot', () => {

        const component = shallow(<NotFound />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()

    })

})
