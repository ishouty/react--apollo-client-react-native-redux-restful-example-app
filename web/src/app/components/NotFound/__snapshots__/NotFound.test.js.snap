// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component should render match snap shot 1`] = `
<div>
  <div
    className="page-not-found-container"
  >
    <Row
      bsClass="row"
      className="main-container"
      componentClass="div"
    >
      <i
        aria-hidden="true"
        className="fa fa-frown-o"
      />
      <h3>
        Opps! Something went wrong. Please try later!
      </h3>
    </Row>
  </div>
</div>
`;
